Add import address function (#3)
rsksmart/btc-transaction-helper: README.md

# btc-transaction-helper

Utility library to generate BTC addresses (legacy, segwit and multisig) and its BTC private key and execute transfers.

> **Note:** These samples require a running bitcoin instance to connect to. Each file contains the configuration that should be updated to point to the running bitcoin instance.

Reviewer: This should propably go below ## Running samples

# btc-transaction-helper

Utility library to generate BTC addresses (legacy, segwit and multisig) and its BTC private key and execute transfers.
## Running samples

> node samples/rpc-commands.js

> node samples/transferBtc-samples.js

> node samples/import-address.js

> node samples/conversion.js
 

> **Note:** These samples require a running bitcoin instance to connect to. Each file contains the configuration that should be updated to point to the running bitcoin instance (except for 'conversion').

Reviewer: ```suggestion
**Note:** These samples (except for 'conversion') require a running bitcoin instance to connect to. Update samples/config.js with the proper configuration.
```

# btc-transaction-helper

> Utility library to generate BTC addresses (legacy, segwit and multisig) and its BTC private key and execute transfers.

Reviewer: ```suggestion
Utility library to interact with a Bitcoin node, generate BTC addresses, execute transfers and parse data.
```

## Running samples

> node samples/rpc-commands.js

> node samples/transferBtc-samples.js

> 'bitcoind' should be running before running these samples.

Reviewer: ```suggestion
**Note:** These samples require a running bitcoin instance to connect to. Each file contains the configuration that should be updated to point to the running bitcoin instance.
```
